src/style.css

```
@@ -1 +1,21 @@
 /* styles go here */
+
+.label {
+  display: block;
+}
+
+.label + .label {
+  margin-top: 10px;
+}
+
+.fieldset + .fieldset {
+  margin-top: 20px;
+}
```

why do you use `+` selector?
would it differ to style just the classnames?
```suggestion
.label {
  margin-top: 10px;
}

.fieldset {
  margin-top: 20px;
}
```